osu.Game.Rulesets.Catch/Objects/Drawables/DrawableBanana.cs

```diff
@@ -44,12 +44,12 @@ protected override void UpdateInitialTransforms()
             const float end_scale = 0.6f;
             const float random_scale_range = 1.6f;
 
-            ScaleContainer.ScaleTo(HitObject.Scale * (end_scale + random_scale_range * RandomSingle(3)))
-                          .Then().ScaleTo(HitObject.Scale * end_scale, HitObject.TimePreempt);
+            this.ScaleTo(HitObject.Scale * (end_scale + random_scale_range * RandomSingle(3)))
```

Comment:
This is probably not what we want to be doing. The rationale behind not adjusting one's own scale is that mods may want to do so as well, causing potentially loss (or otherwise incorrect) transforms.

Can you add back the local scale container and instead forward the interface implementation to the `HitObject` model's scale (can be done implicitly to avoid conflict with the underlying `Drawable.Scale`)?